src/styles/components/_definition.scss

```diff
@@ -46,7 +43,47 @@ $breakpoint-background-image-size: 75rem;
       position: relative;
       font-size: map.get($font-sizes, "sm");
       line-height: map.get($line-heights, "lg"); // approx. 28px/20
-      padding: 2.5rem 8.5625rem;
+      padding: 0 8.5625rem;
+    }
+
+    ul {
+      list-style-type: none;
+
+      li {
+        position: relative;
+        margin: 1rem 0 1rem 1.9rem;
+        font-size: map.get($font-sizes, "xs");
+        line-height: map.get($line-heights, "xl"); // approx. 28px/16
+
+        @media (min-width: $breakpoint-scale-up) {
+          font-size: map.get($font-sizes, "sm");
```

Comment:
These weren't even on my radar as list items. Totally slipped by me. Way to catch them!